services/brig/test/integration/API/User/Client.hs

```diff
@@ -308,10 +309,15 @@ testListClients brig = do
   let (pk1, lk1) = (somePrekeys !! 0, (someLastPrekeys !! 0))
```

Comment:
hlint will soon complain about not using `head` here. Whilst the consistent argument is a solid one, I don't think we want to disable the rule globally.

Reply:
I'm in favor of not applying that hlint rule in tests. In tests we don't have to strive to have total functions and `head` instead of `(!!)`.

Reply:
Fair point. I'll look into disabling the rule for test targets. 👍